fee_allocator/accounting/overrides.py

```diff
@@ -0,0 +1,64 @@
+from abc import ABC, abstractmethod, ABCMeta
+from decimal import Decimal
+from typing import Dict, Type, TYPE_CHECKING
+from fee_allocator.constants import POOL_OVERRIDES_URL
+import requests
+
+if TYPE_CHECKING:
```

Comment:
What does type checking have to do with importing pool fees?  Maybe a comment about what these lines are doing?

Reply:
please read documentation `TYPE_CHECKING` first: https://docs.python.org/3/library/typing.html#typing.TYPE_CHECKING